packages/skills/skills/pikku-build-quick/SKILL.md

```
@@ -0,0 +1,211 @@
+  `.getTime is not a function` at runtime.
+- On SQLite, `db/annotations.ts` is where a `DATETIME` becomes a `Date` and a
+  `JSON` column becomes a typed object. Without an entry they are `string` and
+  `unknown` (PKU481). Add the annotation rather than casting.
+- Surface errors inline next to the control that failed. No empty catch.
+- Every user-facing string is a translation key, not a literal. It is one extra
+  keystroke now and a rewrite later.
+- Never hardcode a host or port — the API base resolves to same-origin `/api`.
+
+Then run it:
+
+    bun run prebuild && bun run dev
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)